staging/velero/templates/miniobackend.yaml

```diff
@@ -0,0 +1,127 @@
+{{- if .Values.minioBackend -}}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: velero-minio-operator
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: minioinstances.miniocontroller.min.io
+spec:
+  group: miniocontroller.min.io
+  version: v1beta1
+  names:
+    kind: MinIOInstance
+    plural: minioinstances
+  scope: Namespaced
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: minio-operator-role
```

Comment:
and here? also heritage info?

Reply:
Yep, good call, forgot. 49f68ee2e0efabb3714e74604b6f866d897316f0 takes care of this.